content/docs/guide/map_submission/map_porting.md

```diff
@@ -9,260 +9,123 @@ weight: 2
+- Mappers can reserve/block porting of their maps using [this form](https://docs.google.com/forms/d/e/1FAIpQLSeheNDY5A960u6GtXCHtt3s_2vZJL3o5tMJ_ZNbYOpb6cx5nQ/viewform).
+- Before porting make sure the map isn't on [this spreadsheet](https://docs.google.com/spreadsheets/d/1KHeWfhGUNpN267CXtPvVdf2h7eQbjPUhWVkE5NimYhg/edit?gid=2051215588#gid=2051215588).
+- When in doubt, contact the mapper ( please don't spam them )
+
+# Setup
+If you have any questions feel free to ask for help in **#map-porting** channel on our [Discord](https://discord.gg/momentummod)!
+
+1. Download [Lumper](https://github.com/momentum-mod/lumper), we will use it to modify the map
+2. Download the map you want to port (maps in **.bz2** format can be extracted using [7zip](https://www.7-zip.org/)):
+    - [fastdl.me](https://main.fastdl.me/69.html) - Contains a huge collection of Surf, Bhop, and KZ maps
+    - [ksf.surf](https://ksf.surf/) (preferred for Surf) - Main hub for competitive surf
+    - [jump.tf](https://jump.tf/forum/) (preferred for RJ/SJ) - Main RJ/SJ forum
+    - [femboy.kz](https://files.femboy.kz/fastdl/csgo/maps/) - CS:GO KZ maps  
+    {{<hint info>}}
+    Porting for gldsrc gamemodes (Conc, HL1Bhop, 1.6KZ) as well as Defrag is way more complicated and requires a different approach.  
+    Please follow the [gldsrc porting guide](/guide/mapping/porting_goldsrc_to_source/) and the [Quake 3 porting guide](/guide/mapping/porting_quake3_to_source/). 
+    {{</hint>}}
+3. Rename the map
+    - Some map names include version info like _a13, _b2, _njv etc. Rename the **.bsp** file to remove them
+    - RJ/SJ maps use **jump_** prefix. This should be changed to **rj_** or **sj_** depending on for which gamemode the map was originally made 
+4. Put the map (**.bsp**) in **/momentum/maps** folder
+    - You can access it by **right clicking** Momentum Mod in your steam library and selecting **Manage → Browse local files**
+5. Open the map using [Lumper](https://github.com/momentum-mod/lumper)
+6. Open the map in Momentum Mod by opening the console (**~** by default, key below ESC) and typing `map <map name>`
+7. Open the console again and enable cheats (`sv_cheats 1`) as well as Lumper synchronization (`mom_lumper_sync 1`)
+8. Click the "Connect to Game Sync" button in Lumper
+
+![Lumper Example](/images/map_porting/connect_to_game_sync.png)
+
+You are now fully set up to start porting the map!
+
+# Porting
+## Step 1: Take Screenshots
+1. Fly around the map in noclip (**g** by default, `noclip` in console) to find good spots to screenshots
+    - You can use maximum of 5 screenshots, 1 for the thumbnail, 4 for the gallery
+2. Use `mom_official_screenshot` in console to take screenshots. This command will automatically apply all relevant settings.  
+    - Screenshots will be saved to **/momentum/screenshots**
+    - You can bind the screenshot command to a key for ease of use in keybind settings or with console by typing `bind <key> mom_official_screenshot`
```

Comment:
I would still include a note about modifying brightness. People are bound to notice this, it makes sense to note why we're doing it.